module/icp/io/aes.c

```diff
@@ -1455,3 +1532,228 @@ aes_mac_verify_atomic(crypto_provider_handle_t provider,
 	return (aes_decrypt_atomic(provider, session_id, &gcm_mech,
 	    key, mac, &null_crypto_data, template, req));
 }
+
+int
+crypto_update_uio_avx(avx_crypt_type_t encrypt, void *ctx, crypto_data_t *input,
+    crypto_data_t *output)
+{
+	common_ctx_t *common_ctx = ctx;
+	zfs_uio_t *uiop = input->cd_uio;
+	zfs_uio_t *ciop = output->cd_uio;
+	off_t offset = input->cd_offset;
+	size_t length = input->cd_length;
+	uint_t vec_idx;
+	size_t cur_len;
+	user_addr_t iov_base = 0ULL, ciov_base = 0ULL;
+	user_size_t iov_len, ciov_len;
+	void* iov_or_mp;
```

Comment:
if this variable is not used, remove it.

Reply:
removed